src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -34,17 +41,31 @@
   end
 end
 
-# If `skip_index` isn't specified, set it to `nothing` and dispatch based on `sparsity_pattern`
-@inline function hadamard_sum!(du, A, volume_flux, orientation, u, equations, sparsity_pattern)
-  return hadamard_sum!(du, A, volume_flux, orientation, u, equations, sparsity_pattern, nothing)
+# Version for dense matrices and non-symmetric fluxes
+@inline function hadamard_sum!(du, A,
+                               flux_is_symmetric::Val{false}, volume_flux,
+                               orientation, u, equations,
+                               sparsity_pattern::Nothing)
+  rows, cols = axes(A)
+  for i in rows
+    u_i = u[i]
+    du_i = du[i]
+    for j in cols
+        AF_ij = A[i,j] * volume_flux(u_i, u[j], orientation, equations)
+        du_i = du_i + AF_ij
```

Comment:
```suggestion
        f_ij = volume_flux(u_i, u[j], orientation, equations)
        du_i = du_i + A[i,j] * f_ij
```
This enables FMAs via `@muladd` and should be slightly faster. It should also be applied to the non-symmetric sparse version below but I can't comment on that one.